tekton_configs/pytest-task.yaml

```diff
@@ -0,0 +1,110 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: pytest
+  namespace: gabrielwen
+  annotations:
+    sidecar.istio.io/inject: "false"
+spec:
+  inputs:
+    params:
+    - name: test-worker-img
+      type: string
+      description: Test worker image.
+      default: gcr.io/kubeflow-ci/test-worker:latest
+    - name: src-dir
+      type: string
+      description: The path for source codes.
+      default: /mnt/testdir/src
+    - name: repos
+      type: string
+      description:
+        Repositories we need to checkout.
+    - name: python-path
+      type: string
+      description:
+        PYTHONPATH env.
+    - name: gcp-sa-dir
```

Comment:
Does this need to a parameter? I think we should try to use as few parameters as possible. 

I think for now you could hard code the secret inside the task to match what's in the cluster. The secret will be a property of the cluster not the individual task run so I don't think it should be a parameter.

i.e. the secret should be configured by the person installing the Task in the cluster not by users consuming a task in a cluster.

The way we can support using different secrets or workload identity would be by using kustomize to generate slightly different versions of the task.